### DISCUSSION TITLE
PDF compression settings?

### DISCUSSION THREAD

**Discussion by ppw0** [cyanfish/naps2#399] [General]
After slightly cropping all pages of a PDF document with many pages, its size increased by five times. I'm not certain of the cause. Is it possible that the settings used by NAPS2 in saving the PDF are too lax, resulting in a PDF that's too large, and is it possible to change these settings?

**Comment by cyanfish** [comment]
If it's a PDF not generated by NAPS2, in order to do image editing like cropping NAPS2 turns each page into an image (rasterization). This can significantly increase the file size, and there's not much that can be done about that within NAPS2 as it's not a full generic PDF editor.

**Comment by ppw0** [comment]
I see. Thanks for the explanation.
